lib/ex_unit/lib/ex_unit/filters.ex

```diff
@@ -214,6 +220,10 @@ defmodule ExUnit.Filters do
     end
   end
 
+  defp has_tag({:line, line}, %{line: _, describe_line: _} = tags, collection) do
+    has_tag({:location, {nil, line}}, tags, collection)
+  end
+
```

Comment:
Not sure what meaning it has when you specify `line:123` and multiple files on the CLI. That will rarely make any sense, no?

Reply:
Instead having :line call :location, I would invert it and make it to so :location calls :line. The rest looks good to me. :)